ui/src/app/views/unregistered/component.spec.ts

```diff
@@ -27,6 +28,27 @@ import {
 } from 'generated';
 
 
+class FailingProfileStub extends ProfileServiceStub {
+  constructor(private fails: number) {
+    super();
+  }
+
+  submitTermsOfService(extraHttpRequestParams?: any): Observable<Profile> {
+    if (this.fails-- > 0) {
```

Comment:
Nit: Spacing

Reply:
I think this spacing was OK, but I agree this looks weird. Split this out onto two lines instead.

Reply:
It was mainly the `-- >` that felt weird. That said, that's cause I was actually reading it as an arrow function rather than a subtraction followed by a > sign. Which doesn't really make sense looking again in context, but I think separating is clearer.

Reply:
Yep, completely agree. Another way might have been `if (0 < this.fails--)` but that felt a bit forced.